src/DearImGui.hs

```diff
@@ -1088,3 +1107,13 @@ popStyleVar n = liftIO do
     m :: CInt
     m = coerce n
   [C.exp| void { PopStyleVar($(int m)) } |]
+
+
+-- | Wraps @ImGui::AddRect()@
```

Comment:
But there's no `AddRect` in `ImGui` :wink: 

This hints at collecting all the `DrawList`-related stuff in its own module as it is a different lower level API.

Reply:
> But there's no `AddRect` in `ImGui` 
> 
> This hints at collecting all the `DrawList`-related stuff in its own module as it is a different lower level API.

Okay, maybe this is in fact a hack to draw a rectangle. Notice that the C code is `GetWindowDrawList()->AddRect`.

Is the `newtype DrawList = DrawList (Ptr ())` correct? This is mimicking the existing `newtype DrawData = DrawData (Ptr ())`.

Reply:
LGTM, but there's no way to use it right now.

Reply:
Should I simply move `addRect` to a new `DrawList.hs` module, or should I actually write the bindings for the `struct ImDrawList`? I don't know how to bind to a struct in C code, where should I look at?

Reply:
`Context`/`destroyContext` appears to demonstrate this:

https://github.com/haskell-game/dear-imgui.hs/pull/44/files#diff-85ba6c168dc51b8fb928344c9e3602f16f3f2026b0e1985ceb688d781daf317eR200